catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpConstants.java

```diff
@@ -14,11 +14,299 @@
 package org.codice.alliance.catalog.transformer.mgmp;
 
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 
+import org.codice.ddf.spatial.ogc.csw.catalog.common.GmdConstants;
+
 public final class MgmpConstants {
```

Comment:
Yep. But since we only have two classes, having them in the same package shouldn't be a problem imho. Unless we need the separate packages from an OSGi bundle perspective.